jdaviz/app.py

```diff
@@ -2319,6 +2320,12 @@ def _on_new_viewer(self, msg, vid=None, name=None, add_layers_to_viewer=False):
         ref_data = self._jdaviz_helper.default_viewer._obj.state.reference_data
         new_viewer_item['reference_data_label'] = getattr(ref_data, 'label', None)
 
+        if (
+                getattr(ref_data, 'meta', {}).get('_WCS_ONLY', False) and
```

Comment:
I had avoided doing that only because of circular imports, originally. But I've pushed [this commit](https://github.com/bmorris3/jdaviz/pull/29/commits/ea13ceacc60678a9fc67ca09d9880d452d268eaf) which imports the function within the method and dodges the circularity.